Source/WebCore/css/values/CSSPrimitiveNumericTypes+ComputedStyleDependencies.h

```diff
@@ -37,11 +37,27 @@ namespace CSS {
 template<> struct ComputedStyleDependenciesCollector<CSSUnitType> { void operator()(ComputedStyleDependencies& dependencies, CSSUnitType); };
 
 // Most raw primitives have no dependencies.
-template<RawNumeric RawType> struct ComputedStyleDependenciesCollector<RawType> { constexpr void operator()(ComputedStyleDependencies&, const RawType&) { } };
+template<RawNumeric RawType> struct ComputedStyleDependenciesCollector<RawType> {
+    constexpr void operator()(ComputedStyleDependencies&, const RawType&)
+    {
+        // Nothing to do.
+    }
+};
 
 // The exception being LengthRaw/LengthPercentageRaw.
-template<> struct ComputedStyleDependenciesCollector<LengthRaw> { void operator()(ComputedStyleDependencies&, const LengthRaw&); };
-template<> struct ComputedStyleDependenciesCollector<LengthPercentageRaw> { void operator()(ComputedStyleDependencies&, const LengthPercentageRaw&); };
+template<auto R> struct ComputedStyleDependenciesCollector<LengthRaw<R>> {
+    inline void operator()(ComputedStyleDependencies& dependencies, const LengthRaw<R>& value)
+    {
+        collectComputedStyleDependencies(dependencies, value.type);
+    }
+};
+template<auto R> struct ComputedStyleDependenciesCollector<LengthPercentageRaw<R>> {
+    inline void operator()(ComputedStyleDependencies& dependencies, const LengthPercentageRaw<R>& value)
```

Comment:
Aren't members with bodies inline by default? Does the keyword do anything here?